### PR TITLE
Users can view a list of all upcoming training programs and create new training programs.

### PR DESCRIPTION
**Pull request by Tommy-Spurlock**
# Description
On the Home page users can click the Training Programs Item to see a list of future training programs. Users my also click the create new button and will see a form to create a new instance of a training program.

## Related Ticket(s)
HR should be able to list and create training programs

## Steps to Test
1. in gitbash cd to banazon-workforce-gnocci
1. start BangazonWorkforce.sln
1. press F5 to run browser
1. in browser click training program. 
1. click create new
1. create new training program who's start date is in the future and press save button. 
1. check list for new instance of training program. 

# Checklist:
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new errors

### CODE REVIEW

**Comment by iamnickmitchell** [comment]
Works as described for me! 🐳🎣

File: BangazonWorkforce/Repositories/TrainingProgramRepository.cs [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+                        reader.Close();
+
+                        return trainingPrograms;
+                    }
+                }
+
+            }
+
+        //public static TrainingProgram GetOneTrainingProgram(int id)
```
Comment (jordan-castelloe) [inline BangazonWorkforce/Repositories/TrainingProgramRepository.cs]: This works as expected, but in general we shouldn't have blocks of commented out code on master. If you want to take notes of functionality to implement later, you can copy all of this to a file of notes and add it to your .gitignore.

**Review by jordan-castelloe — CHANGES_REQUESTED** [review]

File: BangazonWorkforce/Repositories/TrainingProgramRepository.cs (hunk 1 above)
Comment (jordan-castelloe) [reply BangazonWorkforce/Repositories/TrainingProgramRepository.cs]: (but other than that everything looked groovy!)

Comment (Tommy-Spurlock) [reply BangazonWorkforce/Repositories/TrainingProgramRepository.cs]: Okay i'll take it off before I merge. Thanks for the Idea!

**Pull request merged**